tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/cluster/distribution/MessageRedistributionTest.java

```diff
@@ -730,6 +730,40 @@ public void testRedistributionWhenRemoteConsumerIsAdded() throws Exception {
       verifyNotReceive(1);
    }
 
+   @Test
+   public void testRedistributionWhenRemoteConsumerIsAddedLbOff() throws Exception {
+      setupCluster(MessageLoadBalancingType.OFF);
```

Comment:
This test is actually proving we're introducing a breaking change...

Reply:
there are two features in play, the initial load balancing on routing, and redistribution. The fact that they are currently intertwined is a bug, they do different things, this is clear in the code and in the existing doc.

This change separates them further, and what breaks was already broken.
This is my read of what breaks:
enabling redistribution with redistribution-delay != -1 and then using LoadBalanceType.OFF to disable redistribution!.

clearly, to disable redistribution, leave  redistribution-delay at the default of -1. ie: don't configure it in the first place.

The 'broken' case, which is just wrong, can be called out in the release note.

this is a new test that captures the 'now' more clear separation of the features, they are independent and useful when combined.

Reply:
Can we simply just introduce a new setting it is far far safer and better. I see no need to have this as a release note and have users to have breaks when it can be sooo easily avoided with a different setting value